dev-lua/lua-utf8/lua-utf8-0.1.1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="UTF-8 support to Lua"
+HOMEPAGE="https://github.com/starwing/luautf8"
+SRC_URI="https://github.com/starwing/luautf8/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~mips ~ppc ~ppc64 ~x86"
+IUSE="luajit"
+
+RDEPEND="
+	!luajit? ( >=dev-lang/lua-5.1:= )
+	luajit? ( dev-lang/luajit:2 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+DOCS=( README.md )
+
+S="${WORKDIR}/${PN//-/}-${PV}"
+
+src_prepare() {
+	default
+
+	cat > Makefile <<-EOF
+		# this file builds lua-utf8
+
+		LUA_LIBDIR      := ${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_CMOD $(usex luajit 'luajit' 'lua'))
+		LUA_INC         := ${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_INC $(usex luajit 'luajit' 'lua'))
+		LUA_SHARE       := ${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_LMOD $(usex luajit 'luajit' 'lua'))
+		CWARNS          := -Wall -pedantic
+		CFLAGS          += \$(CWARNS) -O3 -I\$(LUA_INC) -fPIC
```

Comment:
`-O3` should not be here,it should come from user's settings. It's generally better to append user flags, than to prepend.

Reply:
fixed